code/game/machinery/ARES/ARES_procs.dm

```diff
@@ -796,18 +853,107 @@ GLOBAL_DATUM_INIT(ares_link, /datum/ares_link, new)
 			current_menu = "maint_claim"
 
 		if("new_report")
-			var/name = tgui_input_text(usr, "What is the type of maintenance item you wish to report?\n\nExample:\n 'Broken light in Aft Hallway.'", "Ticket Name", encode = FALSE)
-			if(!name)
+			var/priority_report = FALSE
+			var/maint_type = tgui_input_list(operator, "What is the type of maintenance item you wish to report?", "Report Category", GLOB.maintenance_categories, 30 SECONDS)
+			switch(maint_type)
+				if("Major Structural Damage", "Communications Failure",	"Power Generation Failure")
+					priority_report = TRUE
+
+			if(!maint_type)
 				return FALSE
-			var/details = tgui_input_text(usr, "What are the details for this report?", "Ticket Details", encode = FALSE)
+			var/details = tgui_input_text(operator, "What are the details for this report?", "Ticket Details", encode = FALSE)
```

Comment:
i know this is existing code but we suuure about skipping encoding? gotta be careful to handle the string safely w/o it

Reply:
It will break people's grammar if encoded 😔 
Has been ok so far.